.github/workflows/node.js.yml

```diff
@@ -43,4 +43,7 @@ jobs:
         npm test
     - name: Run tests on Ubuntu
       if: contains(matrix.os, 'ubuntu')
-      run: npm test
\ No newline at end of file
+      run: npm test
+    - name: Tag
```

Comment:
You need to run the `checkout` step with a PAT, otherwise this won't trigger your downstream `Publish` workflow.

I've added as `PERSONAL_ACCESS_TOKEN`